docs/resources/alert.md

```diff
@@ -51,18 +51,18 @@ EOT
 
 ### Required
 
-- `name` (String)
-- `project_name` (String)
+- `name` (String) Title for the alert
+- `project_name` (String) Name of the project in which to create this alert
```

Comment:
nit: would it be fair to say `Title of the alert`? ffti my compulsion for symmetry